apps/web/.env.example

```diff
@@ -10,8 +10,9 @@ GOOGLE_CLIENT_SECRET=
 
 OPENAI_API_KEY=
 
-UPSTASH_REDIS_URL=redis://localhost:6380
-UPSTASH_REDIS_TOKEN=''
+# local redis config
+REDIS_URL="http://localhost:8079"
+REDIS_TOKEN="27191372ba5c50fd95f59baa43d0e3d2"
```

Comment:
Did you mean to commit this?

Reply:
Yes, we need to set redis_url and redis_token in the .env.example so that when the user 
`cp apps/web/.env.example apps/web/.env` they will automatically have the Redis URL and token set up for their local environment without configuring an Upstash account.

Reply:
Yes, we need to set redis_url and redis_token in the .env.example so that when the user cp apps/web/.env.example apps/web/.env they will automatically have the Redis URL and token set up for their local environment without configuring an Upstash account.